app/assets/javascripts/button_confirmation.js.coffee

```diff
@@ -35,9 +36,16 @@ $ ->
         $('#' + $(this).attr('form')).find('[name=\'_method\']').val 'delete'
       '<p>Are you sure?</p> ' + button + ' <a href="#" class="btn btn-xs btn-primary" id="special_button_color">No, Cancel</a>'
   ).click ->
+    # Bootstrap whitelist: add 'data-method as per bootstrap 3.4.1 update with new sanitizer code
+    # https://getbootstrap.com/docs/3.4/javascript/#js-sanitizer
+    dataMethodWhiteList = $.fn.tooltip.Constructor.DEFAULTS.whiteList
+    originalMethodWhiteList = dataMethodWhiteList.a.slice()
+    dataMethodWhiteList.a.push('data-method')
     t = this
     $('.btn-confirmation').filter(->
       this != t
     ).popover 'hide'
     $(this).popover 'show'
+    # return Bootstrap whitelist to original
+    dataMethodWhiteList.a = originalMethodWhiteList
     false
```

Comment:
Not certain the best way to relax the new sanitation requirements in bootstrap-sass 3.4.1. I've never touched coffeescript before this task so open for suggestions.

The above approach adds the `data-method` then removes from the whitelist. References 
- https://blog.getbootstrap.com/2019/02/13/bootstrap-4-3-1-and-3-4-1/
- https://getbootstrap.com/docs/3.4/javascript/#js-sanitizer

Another option, decided against disabling sanitation for the popup:  
https://github.com/ualbertalib/avalon/commit/eca7f963e0e85e0ac0dbbf361124e3116879583d